scripts_used_in_study/script_used_in_study.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+"""Measure TTL trigger sending latency of several devices.
+
+Required packages:
+
+- numpy
+- pylsl
+- psychtoolbox (https://pypi.org/project/psychtoolbox/)
+
+"""
+
+import os
+import sys
+import time
+
+import numpy as np
+import pylsl
+import psychtoolbox as ptb
```

Comment:
@tstenner these import statements were missing (except pylsl, which I grouped with the other two non-inbuilt imports as is customary) --> did I add them correctly?

Reply:
Sieht gut aus :+1: